### ISSUE TITLE
Consider exposing `children` as a function

### ISSUE THREAD

**Issue by Minitour** [eldaduzman/pymeter#81]
Consider exposing `children` as a function in `BaseThreadGroup` (and other places where applicable).

Sometimes I don't want to pass the children in the `init` because I don't have them yet and would rather add them later.

Current workaround is to do something like this:
```python
tg = ThreadGroupWithRampUpAndHold(10, 1, 60, name="Some Name")
tg._ramp_to_and_hold_instance.children(*[c.java_wrapped_element for c in children])
```

What I would like to see is:
```python
tg = ThreadGroupWithRampUpAndHold(10, 1, 60, name="Some Name")
tg.children(children)
```

**Comment by eldaduzman** [comment]
Hi @Minitour, that's an interesting suggestion.

I've got 2 questions:

1. What do you think should be the behavior in case of repetitive `children` call.

For example:

```python
tg = ThreadGroupWithRampUpAndHold(10, 1, 60, name="Some Name")
tg.children(children1)
tg.children(children2)
```

should the children content be children2? or a concatenation of children1 with children2?

2. What do you think about a property?

For example:

```python
tg = ThreadGroupWithRampUpAndHold(10, 1, 60, name="Some Name")
tg.children = children
```

**Comment by Minitour** [comment]
I think it should be faithful to the original implementation which means to concat.

https://github.com/abstracta/jmeter-java-dsl/blob/master/jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/core/testelements/TestElementContainer.java#L33

**Comment by eldaduzman** [comment]
Make sense, it will be part of the next release.

**Comment by eldaduzman** [comment]
Released in version 0.4
